Inception/ClientApp/src/App/Services.ts

```diff
@@ -0,0 +1,123 @@
+import { LinkTestResult, SiteTestResult } from "./Models";
+import { Observable } from "rxjs/Observable";
+import { Subject } from "rxjs/Subject";
+import { Injectable } from "@angular/core";
+
+@Injectable()
+export class Service {
+
+    public getLinkTestResults(): Observable<LinkTestResult[]>
+    {
+        const subject = new Subject<LinkTestResult[]>();
+        setTimeout(
+            () =>
+            {
+                subject.next(LinkTestResults);
+                subject.complete();
+            },
+            100);
+
+        return subject;
+    }
+
+    public getSiteTestResult(id: string): SiteTestResult
```

Comment:
First letter of method should be in upper case

Reply:
+